### ISSUE TITLE
Should be disabled on egui windows

### ISSUE THREAD

**Issue by pjankiewicz** [johanhelsing/bevy_pancam#3]
I noticed that the drag is captured even when it is done on egui windows

Adding 

```rust
    if egui_ctx.ctx_mut().wants_pointer_input() || egui_ctx.ctx_mut().wants_keyboard_input() {
        return;
    }
```

in systems should fix this.

It can be added behind a feature flag. What do you think about this? I can contribute this change.

**Comment by johanhelsing** [comment]
Yes, that sounds like a good idea!

Sorry for the delayed response.

**Comment by laundmo** [comment]
seconded on this, running into this issue too.

**Comment by johanhelsing** [comment]
Fixed in 899a8381353b725ebf1556072f2403e20044e905

**Comment by laundmo** [comment]
i'm taking a crack at this, though its proven to be difficult due to the usage of Local<>
